invenio_rdm_migrator/streams/communities/load.py

```diff
@@ -8,16 +8,15 @@
 """Invenio RDM migration users load module."""
 
 from ...load import PostgreSQLCopyLoad
-from ...state import CommunitiesState
 from .table_generator import CommunityTableGenerator
 
 
 class CommunityCopyLoad(PostgreSQLCopyLoad):
     """PostgreSQL communities COPY load."""
 
-    def __init__(self, db_uri, data_dir, state, **kwargs):
+    def __init__(self, db_uri, data_dir, state_query, **kwargs):
```

Comment:
```suggestion
    def __init__(self, db_uri, data_dir, state, **kwargs):
```

nit: not crazy about the `_query` suffix. I see the logic but to me it's more of a collection/container object.